### DISCUSSION TITLE
Please trigger a cadano-node-ogmios v6.0.0-rc4_8.6.0-pre docker image

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#348] [Q & A] [answered]
Hello,
I noticed there is no [v6.0.0-rc4_8.6.0-pre](https://hub.docker.com/r/cardanosolutions/cardano-node-ogmios/tags?page=1&name=8.6.0-pre)) docker image with 8.6.0-pre. Can you trigger one, please?
Thank you!

**Comment by KtorZ** [comment, marked as answer]
Hey, not until the integration with 8.6.0 is done. I am fast but 8.6.0 was released this morning at 3am CET. 

So I need a bit of time ☺️
